devices/siglis.js

```diff
@@ -0,0 +1,42 @@
+const fz = require('zigbee-herdsman-converters/converters/fromZigbee');
+const tz = require('zigbee-herdsman-converters/converters/toZigbee');
+const exposes = require('zigbee-herdsman-converters/lib/exposes');
+const reporting = require('zigbee-herdsman-converters/lib/reporting');
+const e = exposes.presets;
+
+
+module.exports = [
+    {
+        zigbeeModel: ['zigfred uno'],
+        model: 'zigfred uno',
```

Comment:
based on the picture on https://www.zigfred.ch/ this seems to should be `ZFU-1a-CH` (not sure if correct, hard to read)

Reply:
`zigbeeModel` is `['zigfred uno']` for sure, as reported by the firmware.
Regarding `model`, I propose to use the human readable model name zigfred uno for a better user experience, ZFU-1D-CH is the internal model name for certification. 
A possibility would be to use `zigfred uno (ZFU-1D-CH)` as `model`.

Reply:
`ZFU-1D-CH` would be a good  `model` (since it yields results when searching for it on google)